src/Libraries/RevitNodes/RevitNodes_DynamoCustomization.xml

```diff
@@ -19,6 +19,9 @@
     <namespace name="Revit.Filter">
       <category>Revit.Filter</category>
     </namespace>
+    <namespace name="Revit.Schedules">
+      <category>Revit.Schedules</category>
+    </namespace>
```

Comment:
@Racel @mjkkirschner Could this be Revit.Views.ScheduleView so it appear with other schedule Node at the same sub category? Should these three nodes be prefixed with ScheduleView for consistency even though this creates long names?